install/terraform/modules/gke/cluster.tf

```diff
@@ -146,7 +146,7 @@ resource "google_container_cluster" "primary" {
 }
 
 resource "google_compute_firewall" "default" {
-  name    = "game-server-firewall-firewall-${local.name}"
+  name    = length(var.firewallName) == 0 ? "game-server-firewall-${local.name}" : var.firewallName
   project = local.project
   network = local.network
 
```

Comment:
Should the firewall open both tcp and udp? I seem to have both rules in my project, but maybe it's because I was testing out the [simple tcp example](https://github.com/googleforgames/agones/tree/master/examples/simple-tcp).

Reply:
I expect most people to only use UDP, so I wouldn't default to more open.